### PR TITLE
Add nr recipe

### PR DESCRIPTION
**Pull request by dfroger**
Note: My goal is to package the [craftr](https://github.com/craftr-build/craftr) build system, this is one of its depenencies.

### CODE REVIEW

**Comment by dfroger** [comment]
During the tests, try to re-build the package (but why?), and fails.

**Comment by scopatz** [comment]
I looks like this can't actually find pandoc on mac.

**Comment by dfroger** [comment]
The package is re-built during the test step, I don't understand why. Maybe this [bug](https://github.com/conda/conda-build/issues/1318)?

File: recipes/nr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+    - pip
+    - setuptools
+    - pandoc
+  run:
+    - python
+
+test:
+  commands:
+    - echo 'skip tests (conda-build bugged)'
```
Comment (scopatz) [inline recipes/nr/meta.yaml]: This is not a good enough test.  There should be an existence test at the very least. see https://github.com/conda-forge/conda-forge.github.io/pull/283/files

**Comment by scopatz** [comment]
Not exactly.  It is failing during the build step.  You are building against Python, so it is succeeding on Python v2.7 and then failing on Python v3.4.  Oddly, though, it is failing because it cannot find pandoc and then it times out.

**Comment by dfroger** [comment]
Sorry, I misread the outputs. I'll see how I can make it pass on Python 3.4, or just skip the pandoc step in `setup.py`. (The fake test was not serious, I'll fix it).

**Comment by dfroger** [comment]
All issues should be now fixed.

**Comment by scopatz** [comment]
This looks good and runs locally. Thanks!

**Pull request merged**